libraries/AP_Scripting/docs/docs.lua

```diff
@@ -1409,6 +1409,17 @@ function AP_Camera__camera_state_t_ud:take_pic_incr() end
 ---@return AP_Camera__camera_state_t_ud|nil
 function camera:get_state(instance) end
 
+---
+-- desc
```

Comment:
```suggestion
-- Change a numbered camera setting to a given value
```

Reply:
ok thanks, I've changed the wording to, "Change a camera setting to a given value"